### PR TITLE
IMP] mail: introduce DropZoneView/isDraggingInside

### PR DESCRIPTION
**Pull request by alexkuhn**
Task-2831082

### CODE REVIEW

File: addons/mail/static/src/models/drop_zone_view.js [hunk 1]
```diff
@@ -15,5 +15,12 @@ registerModel({
             inverse: 'dropZoneView',
             readonly: true,
         }),
+        /**
+         * Determine whether the user is dragging files over the dropzone.
```
Comment (oomsveta) [inline addons/mail/static/src/models/drop_zone_view.js:19]: Determine ➡️ Determine**s**?

File: addons/mail/static/src/models/drop_zone_view.js [hunk 2]
```diff
@@ -15,5 +15,12 @@ registerModel({
...
+         * Useful to provide visual feedback in that case.
+         */
+        isDraggingInside: attr({
+            default: false
```
Comment (oomsveta) [inline addons/mail/static/src/models/drop_zone_view.js:23]: missing trailing comma

File: addons/mail/static/src/models/drop_zone_view.js (hunk 1 above)
Comment (alexkuhn) [reply addons/mail/static/src/models/drop_zone_view.js]: Changes done

File: addons/mail/static/src/models/drop_zone_view.js (hunk 2 above)
Comment (alexkuhn) [reply addons/mail/static/src/models/drop_zone_view.js]: Changes done

**Review by oomsveta — APPROVED** [review]

**Comment by alexkuhn** [comment]
Moved to grouped PR: https://github.com/odoo/odoo/pull/90057

**Pull request closed without merging**